DirectX11/DLLMainHook.cpp

```diff
@@ -228,12 +228,30 @@ static BOOL WINAPI Hooked_SetWindowPos(
     _In_ int cy,
     _In_ UINT uFlags)
 {
-	if (G->SCREEN_FULLSCREEN == 2) {
+	/*
+		TODO: What about this: (dont know how to test it)
```

Comment:
For this, I'm not sure how the hooks will impact your code.

I think it is OK to just go with this, and see if we run into problems.  People using it in the field will report back as they try to use on games that require the hooks.  MGSV is the main game this was created for.

This code suggests it won't work for The Witness, but I'm sure someone will try it.